binding/python/README.md

```diff
@@ -0,0 +1,56 @@
+# Falcon Speaker Diarization Engine
+
```

Comment:
We're a bit inconsistent with the binding headers, but most of our SDKs have the format:
```markdown
# Falcon Binding for Python

## Falcon Speaker Diarization Engine
```